app/mm-redux/actions/users.ts

```diff
@@ -967,7 +865,7 @@ export function startPeriodicStatusUpdates(): ActionFunc {
                     return;
                 }
 
-                getStatusesByIds(userIds)(dispatch, getState);
+                dispatch(getStatusesByIds(userIds));
```

Comment:
This is not a blocking issue and comes only out of curiosity - what motivated this type of refactoring ?

Reply:
The way redux works and my toc for this sort of things